tests/base/consensus_algorithm_01.cc

```diff
@@ -16,7 +16,7 @@
 
 // Test AnonymousConsensusAlgorithmProcess.
 
-#include <deal.II/base/mpi_consensus_algorithm.h>
+#include <deal.II/base/mpi_consensus_algorithm.templates.h>
```

Comment:
Do you need the `.templates.h` here?

Reply:
Since #9604 has been merged not; but generally if one want to use `AnonymousConsensusAlgorithmProcess` on arbitrary types this file also has to be included.

Reply:
I have rebased this branch, instantiate the classes for `unsiged int` here:

https://github.com/dealii/dealii/blob/b831c52623a4c2c5d5061211b29500566e74b661/source/base/mpi_consensus_algorithm.cc#L25-L27

and have reverted the include here.